client/styles/components/_editor.scss

```diff
@@ -79,6 +79,7 @@ pre.CodeMirror-line {
   }
   // left: 0 !important;
   width: #{48 / $base-font-size}rem;
+  z-index: 0;
```

Comment:
We probably don't need an explicit z-index here.

It looks like there's a `z-index: 4` on the `.CodeMirror-gutter-wrapper` div which is coming from the core CodeMirror library.  I assume it's there for a reason so let's not mess with it.